cicecore/cicedynB/dynamics/ice_transport_remap.F90

```diff
@@ -448,17 +453,34 @@ subroutine horizontal_remap (dt,                ntrace,     &
       type (block) ::     &
          this_block       ! block information for current block
 
+#ifdef OMP_TIMERS
+      real (kind=dbl_kind) :: &
+         time0,time1
+#endif
+
       character(len=*), parameter :: subname = '(horizontal_remap)'
 
 !---!-------------------------------------------------------------------
 !---! Remap the ice area and associated tracers.
 !---! Remap the open water area (without tracers).
 !---!-------------------------------------------------------------------
 
+#ifdef OMP_TIMERS
+      time0 = OMP_GET_WTIME()
+
+      if (my_task == master_task) then
+         time1 = OMP_GET_WTIME()
+         write(nu_diag,*) trim(subname),OMP_GET_THREAD_NUM(),iblk,time1-time0,'t1'
+         time0 = time1
+         call flush_fileunit(nu_diag)
+      endif
+#endif
+
       !--- tcraig, tcx, this omp loop leads to a seg fault in gnu
       !--- need to check private variables and debug further
-      !$TCXOMP PARALLEL DO PRIVATE(iblk,ilo,ihi,jlo,jhi,this_block,n,m, &
-      !$TCXOMP          indxinc,indxjnc,mmask,tmask,istop,jstop,l_stop)
+      !$OMP PARALLEL DO PRIVATE(iblk,ilo,ihi,jlo,jhi,this_block,n,   &
+      !$OMP          indxinc,indxjnc,mmask,tmask,istop,jstop,l_stop) &
+      !$OMP SCHEDULE(runtime)
```

Comment:
Does it still lead to a segfault with GCC  ? if not the comment could be removed.

Reply:
Those comment lines are already removed.  Wonder why you're not seeing that.  Check the current file diffs again.

Reply:
Yeah, I realized that after submitting my review, It's because I'm reviewing commit by commit... sorry for the noise.